bot/extensions/fun/weather.py

```diff
@@ -0,0 +1,129 @@
+
+        # Now data_weather contains list of nested dictionaries, check the value of "cod" key is not equal to
+        # "404", means city is found otherwise, city is not found
+        if data_weather["cod"] != "404":
+            # get icon id
+            icon_id = data_weather["weather"][0]["icon"]
+            # store the value of "main" key in variable main
+            main = data_weather["main"]
+            # store the value of visibility into the variable visibility
+            visibility = data_weather['visibility']
+            # store the value corresponding to the "temp" key of main
+            current_temperature = main["temp"]
+            # Convert in Fahrenheit
+            fahrenheit = round((int(current_temperature)) * 1.8 - 459.67,2)
+            celsius = round((int(current_temperature) - 273.15), 2)
+            # store the value corresponding to the "pressure" key of main
+            current_pressure = main["pressure"]
+            # store the value corresponding to the "humidity" key of main
+            current_humidiy = main["humidity"]
+            # store the value of "weather" key in variable forcast
+            forcast = data_weather["weather"]
+            # store the value corresponding to the "description" key at the 0th index of forcast
+            weather_description = forcast[0]["description"]
+            # print following values
+            embed = Embed(
+            color = self.bot.default_color,
+            title = f"{city}",
+                description=f"{timezone_city}",
+            )
+        
+            embed.set_image(
+            url='http://openweathermap.org/img/wn/' + icon_id + '@2x.png'
+            )
+            embed.add_field(
+                name="Description",
+                value=f"{str(string.capwords(weather_description))}"
+                "\u200b",
+                inline=False
+            )
+            embed.add_field(
+                name="Visibility",
+                value=f"{visibility}m | {round(visibility*3.280839895)}ft"
+                "\u200b",
+                inline=False
+            )
+            embed.add_field(
+            name = "Temperature",
+                value=f"{fahrenheit}°F | {celsius}°C"
+                    "\u200b",
+            inline = False
+            )
+            embed.add_field(
+            name = "Atmospheric Pressure",
+            value = f"{current_pressure} hPa"
+                    "\u200b",
+            inline = False
+            )
+            embed.add_field(
+            name = "Humidity",
+            value = f"{current_humidiy}%"
+                    "\u200b",
+            inline = False
+            )
+
+            await ctx.send(embed=embed)
+        else:
+            embeded = Embed(
+                color=self.bot.default_color,
+                # title=f"{city}",
+                description=f"{city.capitalize()} No Found!",
+            )
+            await ctx.send(embed=embeded)
+
```

Comment:
As per the pep8, it is required to have [2 space between top-level functions and classes](https://peps.python.org/pep-0008/#blank-lines).